pkg/kubectl/resource/mapper.go

```diff
@@ -32,7 +32,7 @@ type DisabledClientForMapping struct {
 }
 
 func (f DisabledClientForMapping) ClientForMapping(mapping *meta.RESTMapping) (RESTClient, error) {
-	return nil, nil
+	return nil, fmt.Errorf("attempt to make api call using a disabled client")
```

Comment:
@fabianofranz not sure if this is the right approach, or if it's better to just make and return a fake RESTClient